### PR TITLE
fix: Cache profiles

### PR DESCRIPTION
**Pull request by PhilippeWeidmann**
Depends on #1521

### CODE REVIEW

File: Mail/Views/New Message/Select Mailbox/SelectComposeMailboxViewModel.swift [hunk 1]
```diff
@@ -48,12 +48,17 @@ final class SelectComposeMailboxViewModel: ObservableObject {
     }
 
     func listProfiles() {
```
Comment (adrien-coye) [inline Mail/Views/New Message/Select Mailbox/SelectComposeMailboxViewModel.swift:50]: It looks like the previous version of this method was syncronous. It now hides an async task. Can this be an issue if called from SwUI (glitches …)?

**Review by adrien-coye — COMMENTED** [review]
I only reviewed the code. I did not try to run it.

File: Mail/Views/New Message/Select Mailbox/SelectComposeMailboxViewModel.swift (hunk 1 above)
Comment (PhilippeWeidmann) [reply Mail/Views/New Message/Select Mailbox/SelectComposeMailboxViewModel.swift:50]: Good catch. I ensured we call back on main thread

**Review by adrien-coye — APPROVED** [review]
All good 👍

**Pull request merged**